components/pedal-card-preview/pedal-card-preview.styled.js

```diff
@@ -45,21 +45,19 @@ export const Hint = styled.figcaption`
   text-align: center;
 `;
 
-export const CategoryWrapper = styled.section`
+export const CategoryWrapper = styled.div`
   background-color: transparent;
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 2em;
+  padding: none;
```

Comment:
not sure if `none` is a valid value. Would recommend to just write `0` for "no padding"

Reply:
ou ah, yes, fixed.